src/@chakra-ui/gatsby-plugin/components/Select.ts

```diff
@@ -0,0 +1,68 @@
+import { selectAnatomy } from "@chakra-ui/anatomy"
+import { createMultiStyleConfigHelpers, cssVar } from "@chakra-ui/react"
+import { defineMergeStyles, selectDefaultTheme } from "./components.utils"
+
+const { defineMultiStyleConfig, definePartsStyle } =
+  createMultiStyleConfigHelpers(selectAnatomy.keys)
+
+const $borderBaseWidth = cssVar("border-base-width")
+
+const baseStyleField = defineMergeStyles(selectDefaultTheme.baseStyle?.field, {
+  [$borderBaseWidth.variable]: "1px",
+  borderColor: "currentColor",
+  cursor: "pointer",
+  py: 2,
+  ps: 2,
+  pe: 8,
+  _active: {
+    bg: "primaryLowContrast",
+    boxShadow: "none",
+  },
```

Comment:
Adding active styling is creating a side-effect where on open of the dropdown, a lot of flickering can occur on the options.